x-pack/filebeat/input/httpjson/value_tpl.go

```diff
@@ -128,6 +129,16 @@ func (t *valueTpl) Execute(trCtx *transformContext, tr transformable, defaultVal
 	return val, nil
 }
 
+func tryDebugTemplateValue(target, val string, log *logp.Logger) {
+	banList := map[string]struct{}{
+		"Authorization":       {},
+		"Proxy-Authorization": {},
+	}
+	if _, found := banList[http.CanonicalHeaderKey(target)]; !found {
```

Comment:
```suggestion
	filter := map[string]bool{
		"Authorization":       true,
		"Proxy-Authorization": true,
	}
	if !filter[http.CanonicalHeaderKey(target)] {
```
Though it's probably worth having the map defined outside the scope of the function, or if the list is not going to grow, just a switch with a comma-separated list of filtered targets.
```
    switch http.CanonicalHeaderKey(target) {
    case "Authorization", "Proxy-Authorization":
        // Do not log.
    default:
```
This will be faster for small lists and is non-allocating.